packages/nextjs/app/admin/_components/LargeGrantProposal.tsx

```diff
@@ -93,6 +102,20 @@ export const LargeGrantProposal = ({ proposal, userSubmissionsAmount, isGrant }:
             <ArrowTopRightOnSquareIcon className="w-5 h-5" />
           </Link>
         </div>
+        {(latestStage.approvalVotes.length > 0 || latestStage.rejectVotes.length > 0) && (
+          <div className="flex gap-1 justify-end">
+            {latestStage.approvalVotes.map(vote => (
+              <div key={vote.id} className="tooltip" data-tip={`Pre-approved by ${vote.authorAddress}`}>
```

Comment:
This is showing the address without using the Address component because we need to update to DaisyUI v5 to be able to use tooltip-content https://daisyui.com/docs/v5/

We can improve this later (without updating to DaisyUI v5), resolving the authorAddresses and showing the ENS names (if they have).

Reply:
Create an issue please so we don't forget